### PR TITLE
Put wer tag entity type in SBS output

### PR DESCRIPTION
**Pull request by nishchalb**
The WER tag sidecar file carries accompanying info for tokens in a NLP file. One such type of info is an "entity_type" like NER tags. This type of info is useful to see in the output SBS file.

### CODE REVIEW

**Review by qmac — APPROVED** [review]
LGTM, just looks like that one function needs a refactor

File: src/wer.cpp [hunk 1]
```diff
@@ -525,10 +528,16 @@ void WriteSbs(spWERA topAlignment, string sbs_filename) {
   std::set<std::string> op_set = {"<ins>", "<del>", "<sub>"};
 
   size_t offset = 2;  // line number in output file where first triple starts
-  while (visitor.NextTriple(tk_pair)) {
-    string tk_classLabel = tk_pair->classLabel;
-    string ref_tk = tk_pair->ref;
-    string hyp_tk = tk_pair->hyp;
+  /* while (visitor.NextTriple(tk_pair)) { */
```
[the remark this answers is not in the document]
Comment (nishchalb) [reply src/wer.cpp:531]: It still gets used below

[the remark this answers is not in the document]
Comment (qmac) [reply src/wer.cpp:531]: for the ngrams? goodness that's yucky. I guess at least get rid of the `tk_pair` and `visitor` initialization then

File: src/Nlp.cpp [hunk 2]
```diff
@@ -15,21 +15,36 @@
...
  ************************************/
-NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value normalization)
-    : NlpFstLoader(records, normalization, true) {}
+NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value normalization,
+    Json::Value wer_sidecar)
+    : NlpFstLoader(records, normalization, wer_sidecar, true) {}
 
-NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value normalization, bool processLabels)
+NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value normalization, 
+    Json::Value wer_sidecar, bool processLabels)
```
Comment (dchen579) [inline src/Nlp.cpp:23]: Nit: put nWerSidecar(wer_sidecar) in initializer list instead of line 27

**Pull request merged**